docs/standard/linq/functional-construction-linq-to-xml.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Functional Construction - .NET"
+ms.date: 12/01/2019
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.assetid: 57a82bcf-de03-4f1c-a0c8-9a76e989d542
+---
+# Functional Construction (C# and Visual Basic)
+LINQ to XML provides a powerful way to create XML elements called *functional construction*. Functional construction is the ability to create an XML tree in a single statement.  
+  
+There are several key features of the LINQ to XML programming interface that enable functional construction:  
+  
+- The <xref:System.Xml.Linq.XElement> constructor takes various types of arguments for content. For example, you can pass another <xref:System.Xml.Linq.XElement> object, which becomes a child element. You can pass an <xref:System.Xml.Linq.XAttribute> object, which becomes an attribute of the element. Or you can pass any other type of object, which is converted to a string and becomes the text content of the element.  
+  
+- The <xref:System.Xml.Linq.XElement> constructor takes a `params` array of type <xref:System.Object>, so that you can pass any number of objects to the constructor. This enables you to create an element that has complex content.  
+  
+- If an object implements <xref:System.Collections.Generic.IEnumerable%601>, the collection in the object is enumerated, and all items in the collection are added. If the collection contains <xref:System.Xml.Linq.XElement> or <xref:System.Xml.Linq.XAttribute> objects, each item in the collection is added separately. This is important because it lets you pass the results of a LINQ query to the constructor.  
+  
+These features enable you to write code to create an XML tree. The following is a C# example: 
+  
+```csharp  
+XElement contacts =  
+    new XElement("Contacts",  
+        new XElement("Contact",  
+            new XElement("Name", "Patrick Hines"),  
+            new XElement("Phone", "206-555-0144"),  
+            new XElement("Address",  
+                new XElement("Street1", "123 Main St"),  
+                new XElement("City", "Mercer Island"),  
+                new XElement("State", "WA"),  
+                new XElement("Postal", "68042")  
+            )  
+        )  
+    );  
+```  
+
+These features also enable you to write code that uses the results of LINQ queries when you create an XML tree, as in the following example.
+
+> [!TIP]
```

Comment:
I'd remove this tip

Reply:
I'll remove all of them, though I think they are helpful. The language selector at the top of the page is not obvious (at least it wasn't to me).

At least one existing article has such a tip: [this one](https://docs.microsoft.com/en-us/dotnet/framework/wpf/getting-started/walkthrough-my-first-wpf-desktop-application).